.travis.yml

```diff
@@ -169,11 +176,21 @@ install:
         pip install -q PySDL2;
       fi;
 
+    # Install OSMesa
+    - wget https://github.com/julienr/vispy_osmesa_docker_12_04/releases/download/0.1/osmesa.tar.bz2 -O /tmp/osmesa.tar.bz2
```

Comment:
https://github.com/vispy/demo-data/pull/6

I'll update this url when the demo-data PR gets accepted.